docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -279,212 +216,237 @@ server, DHCP server, RADIUS server) using:
+Creating Authentication Source for Guests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-MAC notification traps
-^^^^^^^^^^^^^^^^^^^^^^
+To keep our example simple, we will simply create a captive portal for guests where they will only have to accept the terms and conditions prior gaining network access. To do so, we must first create a 'Null' authentication source. From 'Configuration->Policies and Access Control->Authentication Sources', click on 'Add Source->External->Null'. As 'Name' and 'Description', specify 'null-source'. Then add an 'Authentication Rules' with name 'catchall' with no condition and with the following 'Actions':
 
-If your switches support MAC notification traps (MAC learnt, MAC removed), we suggest that you activate them in addition to the linkUp/linkDown traps. This way, pfsetvlan does not need, after a linkUp trap, to query the switch continuously until the MAC has finally been learned. When it receives a linkUp trap for a port on which MAC notification traps are also enabled, it only needs to put the port in the MAC detection VLAN and can then free the thread. When the switch learns the MAC address of the device it sends a MAC learnt trap (containing the MAC address) to PacketFence.
+ * Role - guest
+ * Access duratiaon - 12 hours
 
-Port Security traps
-^^^^^^^^^^^^^^^^^^^
+Click on 'Save' to save the new authentication source.
 
-In its most basic form, the Port Security feature remembers the MAC address connected to the switch port and allows only that MAC address to communicate on that port. If any other MAC address tries to communicate through the port, port security will not allow it and send a port-security trap.
+Configure switchport for Web Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If your switches support this feature, *we strongly recommend to use it rather than linkUp/linkDown and/or MAC notifications*. Why? Because as long as a MAC address is authorized on a port and is the only one connected, the switch will send no trap whether the device reboots, plugs in or unplugs. This drastically reduces the SNMP interactions between the switches and PacketFence.
+Connect to that switch over SSH as an admin.
 
-When you enable port security traps you should not enable linkUp/linkDown nor MAC notification traps.
+First, we need to enable Change-of-Authorization (CoA) in our Cisco Catalyst 2960 switch configuration. We essentially need to allow our PacketFence server (172.20.100.2) to send CoA requests to the switch:
 
+  aaa server radius dynamic-author
+   client 172.20.100.2 server-key useStrongerSecret
+  port 3799
 
-Technical introduction to Hybrid enforcement
---------------------------------------------
+Then, we must enable Web Authentication on switch port no. 10. Add the following configuration to the global section:
 
-Introduction
-~~~~~~~~~~~
+  ip device tracking
+  ip http server
+  ip http secure-server
+  snmp-server community public RO
```

Comment:
I think the SNMP must be moved to the initial setup (with 802.1x) since VoIP detection will be attempted and that will cause delays in the request if the SNMP isn't setup properly.

On top of that, we need to tell the user to configure the SNMP communities in the PacketFence switch